### PR TITLE
Add Group Membership Admin to list of admin roles

### PR DESCRIPTION
**Pull request by alifatolahi-okta**
## Description:
- **What's changed?** Add Group Membership Admin to list of admin roles
- **Is this PR related to a Monolith release?** 2020.03.0

### Resolves:

* [OKTA-275744](https://oktainc.atlassian.net/browse/OKTA-275744)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 1]
```diff
@@ -1350,6 +1350,7 @@ Refer to the [product documentation](https://help.okta.com/en/prod/Content/Topic
 | `APP_ADMIN`                   | Application Administrator           | Apps                                  |
 | `USER_ADMIN`                  | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
 | `HELP_DESK_ADMIN`             | Help-Desk Administrator             | [Groups](/docs/reference/api/groups/) |
+| `GROUP_MEMBERSHIP_ADMIN`      | Group-Membership Administrator      | [Groups](/docs/reference/api/groups/) |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: This shouldn't be hyphenated:

> "Group Membership Administrator"

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 2]
```diff
@@ -1350,6 +1350,7 @@ Refer to the [product documentation](https://help.okta.com/en/prod/Content/Topic
 | `APP_ADMIN`                   | Application Administrator           | Apps                                  |
 | `USER_ADMIN`                  | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
 | `HELP_DESK_ADMIN`             | Help-Desk Administrator             | [Groups](/docs/reference/api/groups/) |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: > "Help Desk Administrator"

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 3]
```diff
@@ -1350,6 +1350,7 @@ Refer to the [product documentation](https://help.okta.com/en/prod/Content/Topic
...
+| `GROUP_MEMBERSHIP_ADMIN`      | Group-Membership Administrator      | [Groups](/docs/reference/api/groups/) |
 | `READ_ONLY_ADMIN`             | Read-only Administrator             |                                       |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: should be Read-Only Administrator

Comment (alifatolahi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: changed

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md (hunk 2 above)
Comment (alifatolahi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: changed

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md (hunk 1 above)
Comment (alifatolahi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/roles/index.md]: removed

**Pull request merged**